spec/models/spree/order/state_machine_spec.rb

```diff
@@ -151,44 +159,4 @@
       end
     end
   end
-
-  # Another regression test for Spree #729
```

Comment:
Just wondering why the whole spec was removed?

Reply:
Good question. My judgement was that it was too much effort to rewrite for not much value. But later I deactivated the validation of partial mocks again which makes this spec work. So I re-instated it now. We can solve that problem another time. There's probably still value in that spec if done right. But it may also be covered by other specs already. Right now, it's mocking so much that it's not covering much of the code.